tests/runtime/map

```diff
@@ -1,13 +1,41 @@
+NAME map indexed
+PROG BEGIN { @a[0] = 0; @a[1] = 1; exit(); }
+EXPECT @a[0]: 0
+EXPECT @a[1]: 1
+
+NAME map scalar
+PROG BEGIN { @a = 0; exit(); }
+EXPECT @a: 0
+
+NAME map auto scalar
+PROG BEGIN { @a[0] = 0; exit(); }
+EXPECT @a: 0
+
+# Unfortunately there are some issues when evicting lruhash entries. We can't
```

Comment:
This whole comment should be updated now that we know this is expected behavior and will never be "fixed"

Reply:
Done.